src/Templates/Layout/layout.html

```diff
@@ -1,17 +1,17 @@
 <!DOCTYPE html>
-
 <html>
   <head>
     <meta charset="utf-8"/>
     <title>Control Plane</title>
     <meta name="viewport" content="width=device-width initial-scale="/>
     <base href="/" />
+    <script type="text/javascript" src="/static/main-bundle.1cfb4bf7ac6ca4ccb6da.js"></script>
```

Comment:
Normally it is good practice to have your css link in the header and javascript in the body so you improve time to first contentful paint, but don't block on the javascript which determines time to interactive. Unfortunately (or fortunately?) our CSS is imported via javascript, so if we don't have this in the head then the user will see a flash of the site unstyled, and this is a bit disconcerting.